spec/system/placements/providers/placements/view_placements_list_spec.rb

```diff
@@ -225,6 +226,17 @@
         end
       end
     end
+
+    context "when a user views a placement and returns to the index page" do
+      scenario "User filters are preserved when using the back button" do
+        when_i_visit_the_placements_index_page
+        when_i_check_filter_option("only-available-placements", true)
+        and_i_click_on("Apply filters")
+        and_i_navigate_to("Primary with mathematics")
+        and_i_click_on("Back")
+        then_i_can_see_a_preset_filter("Placements Available", "Placements Available")
+      end
+    end
```

Comment:
Does this change also preserve the search location and page number?

It might be nice to cover those here too, just to be really explicit about it.